docs/content/install/multi-tenant-setup.md

```
@@ -101,9 +120,16 @@ oc get gateway ${TENANT_NAME} -n ${GATEWAY_NAMESPACE}
 ```
 
 !!! tip "Automated script"
-    The `scripts/create-ai-tenant.sh` script automates Gateway, Route, and AITenant creation:
+    The `scripts/create-ai-tenant.sh` script automates Gateway, Route, and AITenant creation.
+    For multi-tenant deployments use the per-gateway label selector and label namespaces manually:
     ```bash
-    ./scripts/create-ai-tenant.sh red-team
+    NAMESPACE_SELECTOR_LABELS="maas.opendatahub.io/gateway-access-red-team=true" \
+      ./scripts/create-ai-tenant.sh red-team
+
+    # Label the infra namespace and any model namespaces after the Gateway is created
```

markdownlint-cli2 (0.23.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)